tests/ci/slack_bot_ci_lambda/app.py

```diff
@@ -1,28 +1,26 @@
 #!/usr/bin/env python3
 
-# A trivial stateless slack bot that notifies about new broken tests in ClickHouse CI.
-# It checks what happened to our CI during the last check_period hours (1 hour) and notifies us in slack if necessary.
-# This script should be executed once each check_period hours (1 hour).
-# It will post duplicate messages if you run it more often; it will lose some messages if you run it less often.
-#
-# You can run it locally with no arguments, it will work in a dry-run mode. Or you can set your own SLACK_URL_DEFAULT.
-# Feel free to add more checks, more details to messages, or better heuristics.
-# NOTE There's no deployment automation for now,
-# an AWS Lambda (slack-ci-bot-test lambda in CI-CD) has to be updated manually after changing this script.
```

Comment:
Please add the name back to the comment. So you will not be the only one who knows how and where we run it